docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx

```diff
@@ -22,7 +22,13 @@ Below is the full list of props that can be used with the `FaceLivenessDetectorC
 
 <ReactPropsTable props={FACE_LIVENESS_DETECTOR_CONFIG} />
 
-### Error States
+### LivenessError
+
+Below is the LivenessError type used in the `onError` function that should give you more granularity into the types of errors happening in the component. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts).
```

Comment:
Is the source code link necessary if `ReactPropsTable>` is there? Just worried we'll break the link unintentionally in the future.

Reply:
Probably not! I guess we included it here to cover all bases but im happy to remove

Reply:
Sounds good!